core/src/test/java/nl/nn/adapterframework/jdbc/dbms/TestBlobs.java

```diff
@@ -11,23 +12,53 @@
 import java.io.Reader;
 import java.io.Writer;
 import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
+import java.sql.SQLException;
 import java.util.function.Consumer;
 
 import org.apache.commons.io.input.ReaderInputStream;
 import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 
+import nl.nn.adapterframework.dbms.JdbcException;
 import nl.nn.adapterframework.jdbc.JdbcQuerySenderBase.QueryType;
-import nl.nn.adapterframework.jdbc.JdbcTestBase;
+import nl.nn.adapterframework.testutil.TestConfiguration;
+import nl.nn.adapterframework.testutil.TransactionManagerType;
+import nl.nn.adapterframework.testutil.junit.DatabaseTest;
+import nl.nn.adapterframework.testutil.junit.DatabaseTestEnvironment;
+import nl.nn.adapterframework.testutil.junit.WithLiquibase;
+import nl.nn.adapterframework.util.JdbcUtil;
 import nl.nn.adapterframework.util.StreamUtil;
 
 @Ignore("Tests for Blobs take too much time and memory to test regularly")
-public class TestBlobs extends JdbcTestBase {
+@WithLiquibase(tableName = TestBlobs.tableName, file = "Migrator/JdbcTestBaseQuery.xml")
+public class TestBlobs {
 
+	protected static final String tableName = "testBlobs_TABLE";
```

Comment:
Static finals zijn meestal UPPER_CASE?
```suggestion
	protected static final String TEST_TABLE = "testBlobs_TABLE";
```